### DISCUSSION TITLE
Drone - dji mavic 2 enterprise advanced

### DISCUSSION THREAD

**Discussion by Tabarane** [bluenviron/mediamtx#1507] [Questions] [answered]
Hi everyone,

has anyone tried to connect drone feeds to "rtsp-simple-server" ->"RSS" , if yes can you share the best configuration for stable video stream and local file record, as im having issues with understanding which stream to provide RSS and im using now "Fussonic media server"as media proxy which takes the rtmp to rtsp in order to do anything, i want to remove this middle man as RSS should be able to do so

**Comment by Tabarane** [comment, marked as answer]
Please ignore, its fixed now, all you have to do is give path in config file without any settings at all, the indicate the server address to send rtmp to, this is all :)

**Reply by rpatidar** [reply]
can mark it answered ?

**Reply by Tabarane** [reply]
yes please this one yes

[a comment by an automated account is not shown]
